debian/clearwater-snmp-handler-alarm.postinst

```diff
@@ -1 +0,0 @@
-postinst
```

Comment:
That's done by the `debian/clearwater-snmp-alarm-agent.postinst` file, isn't it?